src/pages/ProductList/ProductList.jsx

```diff
@@ -1,12 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './ProductList.scss';
+import Product from './component/Product';
+import Filter from './component/Filter';
 
 const ProductList = () => {
+  // 데이터 오는 곳
+  const [isSort, setIsSort] = useState(false);
+
   return (
     <div className="product-list">
-      <div className="container">ProductList</div>
+      <header className="list-header">
+        <h1 className="continent">아시아</h1>
+        <button
+          className="sort-button"
+          onClick={() => setIsSort(prev => !prev)}
+        >
+          <span className="sort-by">정렬 기준</span>
+          {isSort ? (
+            <img
+              className="sort-img"
+              alt="더보기"
+              src="images/icon/angle-up-solid.svg"
+            />
+          ) : (
+            <img
+              className="sort-img"
+              alt="더보기"
+              src="images/icon/angle-down-solid.svg"
+            />
+          )}
+        </button>
+        {isSort ? (
+          <ul className="sort-list">
+            {SORT_MENU.map(menu => (
+              <li key={menu.id}>{menu.content}</li>
+            ))}
+          </ul>
+        ) : null}
```

Comment:
```suggestion
        {isSort && (
          <ul className="sort-list">
            {SORT_MENU.map(menu => (
              <li key={menu.id}>{menu.content}</li>
            ))}
          </ul>
        )}
```

false일 경우 반환하는게 없다면 &&(논리곱 연산자)의 단축평가를 활용 해 보세요.